locales-src/asrouter.ftl

```diff
@@ -81,3 +81,25 @@ cfr-whatsnew-button =
   .tooltiptext = What’s New
 
 cfr-whatsnew-panel-header = What’s New
+
+## Bookmark Sync
+
+cfr-sync-bookmarks-type = Recommended Feature
+cfr-sync-bookmarks-header = Get this bookmark on your phone
+cfr-sync-bookmarks-body = Take your bookmarks, passwords, history and more everywhere you're signed into Firefox.
+cfr-sync-bookmarks-cta-button = Turn on Sync
+
+## Send Tab
+
+cfr-send-tab-type = Recommended Feature
+cfr-send-tab-header-1 = Read this on the go
+cfr-send-tab-header-2 = Take this recipe to the kitchen
+cfr-send-tab-body = Send Tab lets you easily share this link to your phone or anywhere you're signed in to Firefox.
+cfr-send-tab-cta-button = Try Send Tab
+
+## Firefox Send
+
+cfr-firefox-send-type = Recommend Feature
+cfr-firefox-send-header = Share this PDF securely
+cfr-firefox-send-body = Keep your sensitive documents safe from prying eyes with end-to-end encryption and a link that disappears when you're done.
```

Comment:
Apostrophe

Reply:
I've fixed all the apostrophes, I'll ping @Mardak and ask if we can use the variables from brandings.ftl :)

Reply:
I do see the branding strings in our other .ftl files, so it might be fine!

Reply:
`cfr-doorhanger-bookmark-fxa-body` already uses `{ -fxaccount-brand-name }` but that's specially adding the ftl for the bookmark panel:
https://github.com/mozilla/activity-stream/blob/70b6323072f2c801635e453e2e00f2bdb6e1830a/lib/BookmarkPanelHub.jsm#L111

Specifically for CFR page action, I believe additional ftl should be added here:
https://github.com/mozilla/activity-stream/blob/70b6323072f2c801635e453e2e00f2bdb6e1830a/lib/CFRPageActions.jsm#L71